### PR TITLE
fix #83749 nyafilmer2.com

### LINKED ISSUE #83749: nyafilmer2.com
### Issue URL (Ads)
 [https://nyafilmer2.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fnyafilmer2.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3</summary>

  Screenshot 3
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>EasyList<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Adblock Warning Removal List<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion, <br/>AdGuard DNS, <br/>AdGuard Experimental

### PR DESCRIPTION
**Pull request by wout-codes**
#83749

### CODE REVIEW

**Comment by Yuki2718** [comment]
Whoops, missed your PR.

**Comment by wout-codes** [comment]
Fixed the remaining ones.

**Comment by Yuki2718** [comment]
If you add `##.cnotice_holder`, `##.side-info > .close_ad ~ .cnotice_holder` can be removed. I also thought the former is enough.

**Comment by wout-codes** [comment]
This is safer `##.side-info > .close_ad ~ .cnotice_holder`.

**Comment by wout-codes** [comment]
@zloyden What do you think?

**Comment by zloyden** [comment]
> If you add `##.cnotice_holder`, `##.side-info > .close_ad ~ .cnotice_holder` can be removed.

That's true. `##.cnotice_holder` seems ok for this site.

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -587,6 +587,10 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/83749
+nyafilmer.*#$#body { overflow: visible!important; }
+nyafilmer.*#?#body > div[class]:has(> h1:contains(Adblock))
+nyafilmer.*#?#body > div[class]:empty:matches-css(position: fixed):matches-css(z-index: /\d+/)
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:593]: What are these rules for?

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:593]: I am not sure too. I moved it from the foreign files.

Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt:593]: I don't see a detection even with VPN. I guess you can remove those rules.

**Review by zloyden — APPROVED** [review]

**Pull request merged**